<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_win2</class>
 <widget class="QDialog" name="Student_win2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Student_win2{
	
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="8,1,1">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLCDNumber" name="TimelcdNumber">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>90</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExitClass">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="3" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,5">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="VoiceStatusLbl">
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
text-decoration: underline;
font: 13pt &quot;HGF8_CNKI&quot;;
color:rgb(0, 0, 0)
</string>
       </property>
       <property name="text">
        <string>You are not allowed to talk</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopVoicingBtn">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(0, 0, 0);
border-width: 2px;
border-color: rgb(223, 223, 223);
border-style: solid;
</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>978</width>
          <height>648</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>470</height>
         </size>
        </property>
        <widget class="QLabel" name="ShraredScreenArea">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>360</y>
           <width>911</width>
           <height>51</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>984</width>
           <height>728</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                     教师尚未共享屏幕...</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
